azext_iot/central/services/device.py

```diff
@@ -15,10 +15,14 @@
 from azext_iot.central.services import _utility
 from azext_iot.central.models.device import Device
 from azext_iot.central.models.enum import DeviceStatus
+from azure.cli.core.util import should_disable_connection_verify
+
 
 logger = get_logger(__name__)
 
 BASE_PATH = "api/preview/devices"
+http = requests.Session()
```

Comment:
Is there a way to write IT's to validate this behavior?

Is there a way to force a change in the session and verify something breaks?

Is there a way to force the SSL to remain the same and see behavior corrected?

Reply:
request changes on review is for this - all else are nits, feel free to #wontfix them (other than if Paymaun has guidance otherwise)